src/coreclr/clrdefinitions.cmake

```diff
@@ -56,11 +56,12 @@ if(CLR_CMAKE_HOST_WIN32)
   add_definitions(-DWIN32_LEAN_AND_MEAN)
   add_definitions(-D_CRT_SECURE_NO_WARNINGS)
 endif(CLR_CMAKE_HOST_WIN32)
+
 if(CLR_CMAKE_TARGET_WIN32)
   if(CLR_CMAKE_TARGET_ARCH_AMD64 OR CLR_CMAKE_TARGET_ARCH_I386)
-    # Only enable edit and continue on windows x86 and x64
-    # exclude Linux, arm & arm64
+    # Only enable edit and continue on x86 and x64, exclude arm & arm64
     add_compile_definitions($<$<NOT:$<BOOL:$<TARGET_PROPERTY:CROSSGEN_COMPONENT>>>:EnC_SUPPORTED>)
+    add_compile_definitions($<$<NOT:$<BOOL:$<TARGET_PROPERTY:CROSSGEN_COMPONENT>>>:ENC_DELTA_HACK>)
```

Comment:
`ENC_DELTA_HACK` enables ilasm to generate EnC deltas that used to be used for tests.

Do you know whether this hack is still used by EnC tests, or do you plan to use it for hot-reload tests?

If it is still used, I think we be enabled using the same condition as before. I assume we will want to use it for hot-reload tests too. 
If it is not used, we should just delete it.

Reply:
I don't plan to use it for hot reload testing and the ENC tests in the debugger test repo are not enabled. I don't think Roslyn's ENC testing use it because I think they don't use ilasm to generate deltas. I think we will need a better way to generate the diffs for both ENC and hot reload tests. @hoyosjs and @davmason do you think we will use ildasm to generate diffs in the long term?  I'll remove the ENC_DELTA_HACK code if they agree.

I wanted to maintain parity on Windows just in case we need it or use it, but not on xplat mainly because the ifdef'ed code doesn't build on xplat and I didn't think it was worth the time to get it to build.

Reply:
No currently running tests use ilasm to do EnC testing, and long term we should be moving away from it. We should be using roslyn which has the benefit of testing the actual end to end scenario, and lets us write in C# instead of IL.

My vote is for getting rid of ENC_DELTA_HACK.

Reply:
I removed all the ENC_DELTA_HACK code.